cmd/exec.go

```diff
@@ -81,7 +86,13 @@ var execCmd = &cobra.Command{
 			if cont.State != "running" {
 				continue
 			}
-			stdout, stderr, err := c.Runtime.Exec(ctx, cont.ID, cmds)
+
+			nodeRuntime, err := c.GetNodeRuntime(cntName)
+			if err != nil {
+				log.Fatal(err)
```

Comment:
should we return the error instead of log.Fatal?

Reply:
yep